AnalyzerTools/src/GEScaleSyst.C

```diff
@@ -0,0 +1,2411 @@
+#ifndef HighPtMuonAnalysis_GEScale
+#define HighPtMuonAnalysis_GEScale
+
+#include "GEScaleSyst.h"
+
+using namespace std;
+
+ScaledPts GEScaleSyst::GEPt(int Year, float pt, float eta, float phi, int charge){
+
+  ScaledPts out;
+  out.ScaledPt = pt;
```

Comment:
why are lines 11-13 needed? They seem to be overwritten in lines 32-34.

Reply:
It's a copy of Min's repo and I don't wanna touch it